### ISSUE TITLE
Questionable Ninja Names

### ISSUE THREAD

**Issue by DummeHytteOst** [space-wizards/space-station-14#28323]
## Description

There exist at least two combinations of ninja names and titles, that are very questionable, one is just straight up in violation of the no ERP rule, the other is very questionable.
**Reproduction**

Get unlucky and roll either 

1. Deep as your name and Throat as your title.

2. Black as your name and Bro as your title.

**Screenshots**

the players name have been blocked due to privacy and are unrelated.
![image](https://github.com/space-wizards/space-station-14/assets/94384139/3e695bbd-ac0e-4605-98ec-511141e020a6)

i do not have a screen shot of the second combination, but it is possible to get.

**Comment by DummeHytteOst** [comment]
whoops, i think the discord auto mod thingy removed the issue in the #github channel 😬

**Comment by superjj18** [comment]
Ss14 players when color, no idea why people treat black like it’s a slur

Deep Throat ain’t that bad either considering its historical significance as the name of the secret informant in the watergate scandal(or Gray Fox’s code name in Metal Gear Solid). Spy-esque lol

**Comment by GroggleG** [comment]
Yeah I dont understand why people care?
Even if they werent references like get over it, black is there because black rain?
black snake?
Its not a big deal, why bother.

**Comment by DummeHytteOst** [comment]
> Ss14 players when color, no idea why people treat black like it’s a slur

i don't think anyone treats black like its a slur, something like black dragon is a standard generic ninja esque name,
but people found it uncomfortable being called Black Bro, or calling someone Black Bro, hence why mentioned it in the issue.
 
>Deep Throat ain’t that bad either considering its historical significance as the name of the secret informant in the watergate scandal(or Gray Fox’s code name in Metal Gear Solid). Spy-esque lol

It might be a reference to some informant, but people should not be named after a sex act on a 16+ server with minors, especially when its forced, and there is no way around the fact that it is against the rules, rule 3 clearly states "This includes direct and indirect mentions of sexual behavior or actions."

**Comment by GroggleG** [comment]
> > Ss14 players when color, no idea why people treat black like it’s a slur
> 
> i don't think anyone treats black like its a slur, something like black dragon is a standard generic ninja esque name, but people found it uncomfortable being called Black Bro, or calling someone Black Bro, hence why mentioned it in the issue.
> 
> > Deep Throat ain’t that bad either considering its historical significance as the name of the secret informant in the watergate scandal(or Gray Fox’s code name in Metal Gear Solid). Spy-esque lol
> 
> It might be a reference to some informant, but people should not be named after a sex act on a 16+ server with minors, especially when its forced, and there is no way around the fact that it is against the rules, rule 3 clearly states "This includes direct and indirect mentions of sexual behavior or actions."

While the last one is true, the others are simply a issue of being "uncomfortable"
Almost every single name in the name gen is a popculture or espionage reference, if you want simply remove throat as a name given it is the most localized and does not offer much for name combination.

**Comment by Just-a-Unity-Dev** [comment]
"Deep Throat" is against the rules under one of the no ERP rules (we have minors) for obvious reasons

**Comment by GroggleG** [comment]
> "Deep Throat" is against the rules under one of the no ERP rules (we have minors) for obvious reasons

Agreed, either recode the component to blacklist combinations or get rid of either Throat or both Deep and Throat.

**Comment by Kadeo64** [comment]
as much as I find the fact richard nixon's horrible choices breaking ss14 rules extremely funny I think that removing the throat part entirely is probably better in the long run.

**Comment by engineer-pearl** [comment]
What about allowing the ninja to reroll a name before spawning? Auto generated names are a place where you're likely to get a few surprises, so being able to handle a bad roll could be useful.

**Comment by Luxzhv** [comment]
Someone could write a check for these two instances, and have them force rerolled, to something else. Lil if statement maybe.

**Comment by GroggleG** [comment]
> What about allowing the ninja to reroll a name before spawning? Auto generated names are a place where you're likely to get a few surprises, so being able to handle a bad roll could be useful.

Eh that just removes variety, you get what you get and if you wanted something different better luck next time.

**Comment by deltanedas** [comment]
just remove deep throat 99% of people will know it as :trollface: and not the informant

**Comment by GroggleG** [comment]
> just remove deep throat 99% of people will know it as :trollface: and not the informant

I have said this atleast twice now :death:
